NEWS.md

```diff
@@ -0,0 +1,26 @@
+# data.validator 0.2.0
+
+-   `validate_cols()` will use all columns in dataframe if no column is passed
```

Comment:
`validate_rows` was also changed in the same PR: https://github.com/Appsilon/data.validator/pull/53